pkg/convert/xliff12_test.go

```diff
@@ -18,27 +18,27 @@ import (
 
 // TODO: XLIFF1.2 and XLIFF2.0 uses same test data and same tests, so we can merge them into one test file
 
-// randXliff12 dynamically generates a random XLIFF 1.2 file from the given messages.
-func randXliff12(messages *model.Messages) []byte {
+// randXliff12 dynamically generates a random XLIFF 1.2 file from the given translations.
```

Comment:
// randXliff12 dynamically generates a random XLIFF 1.2 file from the given translation.